app/models/component/manifest/app_container.rb

```diff
@@ -37,6 +37,11 @@ def environment_settings
         env << { name: '_JAVA_OPTIONS', value: '-Xmx1280m -Xms1280m' }.with_indifferent_access
         env << { name: '_BUSBAR_BUILD_TIME', value: timestamp.to_s }.with_indifferent_access
         env << { name: 'PORT', value: app_port.to_s }.with_indifferent_access
+        env << { name: 'DATADOG_LOGS_NAME', value: app_id}.with_indifferent_access
```

Comment:
To be consistent with previous lines, it looks like we would need to add a space to the end of the hash before the closing curly brace like so `app_id }`

Reply:
applies to lines 40-44

Reply:
Fixed.